src/components/ActionModal/ActionOptionsModal.js

```diff
@@ -19,84 +19,76 @@
 */
 import * as React from 'react';
 import { SafeAreaView } from 'react-navigation';
-import styled, { withTheme } from 'styled-components/native';
+import styled from 'styled-components/native';
 import SlideModal from 'components/Modals/SlideModal';
+import { BaseText } from 'components/Typography';
 import ListItemWithImage from 'components/ListItem/ListItemWithImage';
-import { getThemeColors, themedColors } from 'utils/themes';
+import { themedColors } from 'utils/themes';
 import { spacing } from 'utils/variables';
-import type { Theme } from 'models/Theme';
 
 type ItemType = {
   label: string,
   key: string,
+  iconName?: string,
   onPress: () => void,
   value?: string,
   isDisabled?: boolean,
   hide?: boolean,
 }
 
 type Props = {
-  theme: Theme,
-  onModalClose: (?() => void) => void,
+  onModalClose: (Function) => void,
   isVisible: boolean,
   items: ItemType[],
   doNotCloseOnPress?: boolean,
+  title?: string,
 };
 
-type ItemProps = {
-  label: string,
-  onPress: Function,
-  value: string,
-  isDisabled?: boolean,
-  iconName?: string,
-};
-
-
 const MainContainer = styled.View`
-  padding: 20px ${spacing.layoutSides}px 30px;
+  padding: 22px ${spacing.layoutSides}px 50px;
```

Comment:
Seems like horizontal paddings are still too large. 
It's 30px in design.
SlideModal has a 16px padding by default. So either here horizontal padding should be set to 0, and ListItem horizontal paddings should be set to 14px or wise versa :)

As for bottom padding - designs are showing "safe space" padding (as they are designed on iPhone). In action we do not need such a long padding - it is being handled by SafeArea. (You can check how we are handling this on EventDetails modal :) )